content/10.blackbox.md

```diff
@@ -1,17 +1,15 @@
 ## Tip 8: Your DL models can be more transparent {#blackbox}
 
-In ML, interpretability refers to the study of the discriminative features used for classification or regression task.
-ML models can vary in terms of interpretability from a “transparent” to a “black-box” model, the first with a clear description of features importance found, for example, in common random forests implementations [@doi:10.1023/A:1010933404324]. 
-The second for the most widely used DL implementations.  
-Because of the large number of parameters and non-linear relationships among features, DL models are hard to interpret when compared to other ML models. 
+In ML, a model's interpretability refers to the ability to identify the discriminative features used for learning and that influence or sway the predictions.
+ML models vary widely in terms of interpretability, ranging from being fully transparent to being a “black-box”.
+In the former case, a clear description of the features' importance can found, such as with logistic regression or decision trees. 
+However, DL models are harder to interpret compared to other ML models because of their large number of parameters and the non-linear relationships between their features.
 
-There are, however, many strategies to interpret a DL models.
-For example, autoencoders (AE) is a family of unsupervised methods that aim to learn a new (encoded) representation and minimize the error between the new representation and the input data. 
-Tan et al., [@doi:10.1142/9789814644730_0014] used a denoising AE to summarize key features from breast cancer dataset.
-The authors could map encoded features to clinical characteristics relevant to the disease. 
+Model interpretability is notably important in the biomedical field, where the decisions of models can mean the difference between life and death.
+Many authors attribute the lack of pervasiveness of DL tools in healthcare because of the inability to understand what these models learn [@doi:10.1109/JBHI.2016.2636665; @doi:10.1038/s41551-018-0315-x]. 
+Luckily, there are many strategies to interpret both ML and DL models.
+In fact, due to the increasing demand for interpretable models, the literature on the topic is currently growing at an exponential rate [@arxiv:2001.02522].
+Therefore, instead of recommending specific methods for either DL-specific or general-purpose model interpretation, we suggest consulting a recent overview.
```

Comment:
```suggestion
Therefore, instead of recommending specific methods for either DL-specific or general-purpose model interpretation, we suggest consulting [@url:https://christophm.github.io/interpretable-ml-book/] which is freely available and continually updated.
```